opslevel/datasource_opslevel_domain.go

```diff
@@ -24,8 +24,39 @@ type DomainDataSource struct {
 	CommonDataSourceClient
 }
 
-// DomainDataSourceModel describes the data source data model.
-type DomainDataSourceModel struct {
+var domainDatasourceSchemaAttrs = map[string]schema.Attribute{
+	"id": schema.StringAttribute{
+		MarkdownDescription: "The ID of this Doamin",
+		Computed:            true,
+	},
+	"aliases": schema.ListAttribute{
+		ElementType: types.StringType,
+		Description: "The aliases of the domain.",
+		Computed:    true,
+	},
+	"name": schema.StringAttribute{
+		Description: "The name of the domain.",
+		Computed:    true,
+	},
+	"description": schema.StringAttribute{
+		Description: "The description of the domain.",
+		Computed:    true,
+	},
+	"owner": schema.StringAttribute{
+		Description: "The id of the domain owner - could be a group or team.",
```

Comment:
```suggestion
		Description: "The id of the domain owner (team)",
```

I recommend we also CTRL+F for `group` in this repo and delete all mentions of it.